app/actions.js

```diff
@@ -713,6 +722,13 @@ const initializeCabal = ({ addr, isNewlyAdded, username, settings }) => async di
         dispatch(onIncomingMessage({ addr, channel, message }))
       }
     }, {
+      name: 'private-message',
+      throttleDelay: 500,
+      action: (data) => {
+        console.log('private-message', data)
+      }
+    },
+    {
```

Comment:
reckon this should probably be handled more than just a console.log? or is that enough to trigger a state change?

Reply:
it's not used at all. the `new-message` event is all that's needed for handling new private messages. we can pull this out but i left it in as a reminder that that event is a thing

Reply:
ah ok! :)